src/genie/libs/parser/iosxe/tests/ShowLispEidTableVrfUserIpv4MapCache/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,571 @@
+expected_output = {
+    "vrf": "User",
+    "iid": 4100,
+    "number_of_entries": 2186,
+    "eid": {
+        "0.0.0.0/0": {
+            "subnet": "/0",
+            "uptime": "1w6d",
+            "expire": "never",
+            "source": [
+                "static-send-map-request"
+            ],
+            "rloc": {
+                "status": "negative_cache_entry",
+                "action": "send-map-request"
+            }
+        },
+        "0.0.0.0": {
```

Comment:
any specific reason why subnet was removed from `0.0.0.0`? if not, let's have like below and no need `subnet`:
```
    "0.0.0.0/5": {
        "uptime": "4w6d",

Reply:
There was no specific reason. I went ahead and changed it to your suggestion. Thanks!